app/lists/template.hbs

```diff
@@ -0,0 +1,14 @@
+<nav>
+  <div class="list-list">
+    <ul class="sub-nav-list">
+      {{#each model as |list|}}
+      {{#link-to "lists.reminders"}}
```

Comment:
Put the `link-to` with in the `li`, also `lists.reminders` needs to know what to fill in for the dynamic route segment https://guides.emberjs.com/v2.5.0/templates/links/#toc_the-code-link-to-code-helper